notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
     "This notebook demonstrates style transfer with OpenVINO, using the Style Transfer Models from [ONNX Model Repository](https://github.com/onnx/models), specifically [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) model is one of the style transfer models designed to mix the content of an image with the style of another image. \n",
     "\n",
-    "<img align='center' src=\"https://user-images.githubusercontent.com/109281183/204767232-95b8f0a8-010e-4f6d-981e-104200652098.jpg\" alt=\"drawing\" width=\"1000\"/>\n",
+    "<img align='center' src=\"https://user-images.githubusercontent.com/109281183/208703143-049f712d-2777-437c-8172-597ef7d53fc3.gif\" alt=\"drawing\" width=\"1000\"/>\n",
```

Comment:
I recommend using Markdown instead of HTML. The same as you did in the README file :)

Reply:
Thanks for the review. I have updated as requested using Markdown for the image.